packages/admin-stories/src/docs/components/ClearInputButton/stories/ClearableInputField.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import { ClearInputButton } from "@comet/admin";
+import { Box, Input, Typography } from "@material-ui/core";
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+
+storiesOf("stories/components/Clear Input Button/Clearable Input Field", module).add("Clearable Input Field", () => {
+    const [inputText, setInputText] = React.useState<string>("");
+
+    return (
+        <Box display={"flex"} alignItems={"center"}>
+            <Box marginRight={15}>
+                <Typography variant={"body1"}>Input Field with clearable onClick Button Functionality:</Typography>
+            </Box>
+            <Input
```

Comment:
Please use `InputBase` instead of `Input`, we don't really want to use the Mui-Style Input anywhere as it does not match the Comet-CI.